lib/requesters/appsMappedRequests.js

```diff
@@ -32,19 +40,10 @@ async function processFullDetailApps (apps, opts) {
   return Promise.all(promises);
 }
 
-async function processAndRecur (html, opts, savedApps, mappings) {
-  if (R.is(String, html)) {
-    html = scriptData.parse(html);
-  }
-
-  const processedApps = appList.extract(mappings.apps, html);
-  const apps = opts.fullDetail
-    ? await processFullDetailApps(processedApps, opts)
-    : processedApps;
-  const token = R.path(mappings.token, html);
-
-  return checkFinished(opts, [...savedApps, ...apps], token);
-}
+const REQUEST_MAPPINGS = {
```

Comment:
Do you believe that adding this closer to where this is used will be better for maintenance?

Reply:
yes I want to reduce module surface (fewer deeper modules instead of multiple shallower ones), especially for things that are used in a single place I'd rather have them in a single spot. 

(I haven't finished this PR btw)